### ISSUE TITLE
[Carousel]: WCAG problems

### ISSUE THREAD

**Issue by mart-sessman** [TEDI-Design-System/angular#241]
## Problem
* Hidden slides are being read by screenreader
* Indicator size (8x22 px) does not meet EN 301 549 criteria 9.2.5.8 (24x24px)
## Solution
Slides not visible should be aria-hidden="true"
Increase indicator size to meet the criteria
